components/style/values/mod.rs

```diff
@@ -114,6 +114,29 @@ impl ToCss for CustomIdent {
     }
 }
 
+/// Nonstandard (https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-context-properties)
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToComputedValue, ToCss)]
+pub struct MozContextProperties(pub CustomIdent);
```

Comment:
Yeah, this could be in `svg.rs`, and you'd only need to export it from `values/mod.rs`.